### PR TITLE
Support windows file path & env vars for CLI serve command

### PR DESCRIPTION
**Pull request by axellaurelut**
Hello,

Currently, running `dispatch server develop` fails on windows because of the static file paths.
Also the front-end will never be served  because the env var `STATIC_DIR` has a default.

### CODE REVIEW

**Comment by mvilanova** [comment]
Thanks for the PR. I'll test the changes locally and merge them if everything is working as expected.

File: src/dispatch/cli.py [hunk 1]
```diff
@@ -697,15 +697,20 @@ def run_server(log_level):
...
 
         # take our frontend vars and export them for the frontend to consume
         envvars = os.environ.copy()
         envvars.update({x: getattr(config, x) for x in dir(config) if x.startswith("VUE_APP_")})
 
-        p = Popen(["npm", "run", "serve"], cwd="src/dispatch/static/dispatch", env=envvars)
+        p = Popen(
+            ["npm", "run", "serve"],
+            cwd=os.path.join("src", "dispatch", "static", "dispatch"),
+            shell=True,
```
Comment (mvilanova) [inline src/dispatch/cli.py:711]: Adding `shell=True` breaks my local dev env. It should be working without it in Windows according to the [Popen docs](https://docs.python.org/3/library/subprocess.html#popen-constructor):
> On Windows with shell=True, the COMSPEC environment variable specifies the default shell. The only time you need to specify shell=True on Windows is when the command you wish to execute is built into the shell (e.g. dir or copy). You do not need shell=True to run a batch file or console-based executable.

Also, there are [security considerations](https://docs.python.org/3/library/subprocess.html#security-considerations) that need to be taken into account when using `shell=True`.

Can you try without it and let me know if it works for you?
```suggestion
```

**Review by mvilanova — CHANGES_REQUESTED** [review]

**Comment by axellaurelut** [comment]
Removing `shell=True` results on a `FileNotFoundError: [WinError 2] The system cannot find the file specified` error. Looks like a known issue (https://bugs.python.org/issue20927).

There is a work around though (https://stackoverflow.com/questions/49018413/filenotfounderror-subprocess-popendir-windows-7/49018557), I updated the PR, let me know if that is OK for you.

**Review by mvilanova — APPROVED** [review]
Changes look good.

**Pull request merged**